conf/application.conf

```diff
@@ -107,4 +111,5 @@ tracking-consent-frontend {
 
 features {
   welsh-translation: true
+  ema-supporting-agents-enabled = true
```

Comment:
Minor change on rebase. This has been renamed to feature-switch to fix some config issues picked up. Just double check any config added calls feature-switch instead of features after rebase